tests/selectedtests/app/conftest.py

```diff
@@ -1,16 +0,0 @@
-from unittest.mock import MagicMock
-
-import pytest
-
-from selectedtests.app import app
-
-
-@pytest.fixture()
-def app_client():
-    """
-    Client for the flask web with mocked endpoints.
-    """
-    mongo = MagicMock()
-    evg_api = MagicMock()
-    client = app.create_app(mongo, evg_api).test_client()
-    return client
```

Comment:
Mypy didn't like the duplicate module name, so I moved this into the other conftest.